src/renderer/auth/Cloud.js

```diff
@@ -226,31 +222,30 @@ export class AuthAccess {
         DEV_ENV &&
           rtv.verify(
             { idpClientId: normalizedNewValue },
-            { idpClientId: AuthAccess.specTs.idpClientId }
+            { idpClientId: Cloud.specTs.idpClientId }
           );
         _changed = _changed || _idpClientId !== normalizedNewValue;
         _idpClientId = normalizedNewValue;
       },
     });
 
     /**
-     * @member {boolean|null} usesSso True if access is via SSO; false if it's via
-     *  basic auth; null if unknown
+     * @member {string|null} cloudUrl
      */
-    Object.defineProperty(this, 'usesSso', {
+    Object.defineProperty(this, 'cloudUrl', {
       enumerable: true,
       get() {
-        return _usesSso;
+        return _cloudUrl;
       },
       set(newValue) {
         const normalizedNewValue = newValue ?? null; // undefined -> null
```

Comment:
And then this should change to
```suggestion
        const normalizedNewValue = newValue || null; // '' or undefined -> null
```
so that we also normalize empty strings to null since an empty string means nothing here. It's either null or a non-empty string. Before, it was `usesSso` and it was a boolean, which had different normalization requirements because a boolean is either true or false (and if not set, we'd normalize to null).

Reply:
fixed